infrastructure/aat.tfvars

```diff
@@ -1,9 +1 @@
-capacity = "2"
-idam_api_url = "https://idam-api.aat.platform.hmcts.net"
-idam_client_redirect_uri = "https://rpe-bulk-scan-processor-sandbox.service.core-compute-sandbox.internal/oauth2/callback"
 supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE", "CMC"]
```

Comment:
why do you keep these?

Reply:
It's used to build following vault keys:idam_users_usernames / idam_users_passwords, ie idam-users-cmc, etc

If that's not used later I am happy to remove but probably it's going to delete those keys from vault!